okio/src/jvmTest/java/okio/ByteStringJavaTest.java

```diff
@@ -92,7 +93,7 @@
         new Object[] { Factory.ONE_BYTE_PER_SEGMENT, "SegmentedByteString (one-at-a-time)" });
   }
 
-  @Parameter(0) public Factory factory;
+  @Parameter() public Factory factory;
```

Comment:
I think this is more clear with the literal 0

Reply:
Reverted.